extensions/wikia/ImageReview/ImageReviewDatabaseHelper.class.php

```diff
@@ -23,15 +23,16 @@ public function selectImagesForList( $sOrder,
 		$oDB = $this->getDatawareDB( DB_SLAVE );
 
 		$oResults = $oDB->query('
-			SELECT pages.page_title_lower, image_review.wiki_id, image_review.page_id, image_review.state, image_review.flags, image_review.priority, image_review.last_edited
```

Comment:
The only caller of this method does not make any use of `page_title_lower` field.